docs/Stryker-in-pipeline.md

```diff
@@ -48,3 +48,40 @@ Example for azure devops with dashboard storage provider:
 dotnet stryker --dashboard-compare --baseline-storage-location Dashboard --dashboard-api-key $(Stryker.Dashboard.Api.Key)--dashboard-version $(System.PullRequest.SourceBranch) --git-source $(System.PullRequest.TargetBranch)
 dotnet stryker -compare -bsl Dashboard -dk $(Stryker.Dashboard.Api.Key)-version $(System.PullRequest.SourceBranch) -source $(System.PullRequest.TargetBranch)
 ```
+
+## Azure DevOps Extensions
+There are multiple Azure DevOps extensions to enrich the Stryker experience. 
+
+### Mutation Report Publisher
+With this extension you add the mutation report to your build summary so you can view the mutation test result on azure devops.
+
+In YAML pipelines add the following after the test step:
+
+```yaml
+- task: PublishMutationReport@0
+  displayName: 'Publish Mutation Test Report'
+  inputs:
+    reportPattern: '**/mutation-report.html'
+```
+
+You now get a new tab on your build pipeline where you can click through the complete mutation testing report generated by your mutation test run. When using Stryker the [mutation testing report](https://github.com/stryker-mutator/mutation-testing-elements) looks like:
+
+![mutation report in a build](./images/devops-report-publisher-tab.png)
+
+To install 'Mutation Report Publisher' or for more information go to the [marketplace](https://marketplace.visualstudio.com/items?itemName=stryker-mutator.mutation-report-publisher).
+
+### Run Stryker .Net
+**Disclaimer**: This is not an official [Stryker Mutator](https://stryker-mutator.io/) extension. 
```

Comment:
This extension uses powershell so is not cross platform. It works on windows only.